neuvector-scanner.yaml

```diff
@@ -59,6 +61,33 @@ subpackages:
           mkdir -p ${{targets.contextdir}}/usr/local/bin
           install -Dm755 monitor/monitor ${{targets.contextdir}}/usr/local/bin/monitor
 
+  - name: neuvector-db
```

Comment:
I see @EyeCantCU is on the reviews, but I recall an intentional decision to remove this in the past

Reply:
> I see @EyeCantCU is on the reviews, but I recall an intentional decision to remove this in the past

@mamccorm  To _remove_ this, or to move it to a separate package? We currently do have a separate `neuvector-db` package: this change will ensure that it remains current.

Reply:
Hey Kyle. I would pull the database personally. It's not consumed by any of our images today, and it's likely to lag behind whatever database is provided in the latest image as it's updated constantly